src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -41,6 +41,11 @@ public class DefaultServiceControl : IServiceControl
         IEventAggregator eventAggregator;
         ProfilerSettings settings;
 
+        static DefaultServiceControl()
+        {
+            ServicePointManager.ServerCertificateValidationCallback = (sender, certificate, chain, errors) => ApplicationConfiguration.SkipCertificateValidation;
```

Comment:
The problem with this is that every single time the certificate is checked, it will call back into app.config. As I said above, I believe Program.cs should read the app.config value and set this property at app start up.

Reply:
Agreed. There's no Program.cs so moved out to AppBootstrapper that executes once on app_startup.